File: Motocitizen/src/motocitizen/network/Methods.java
package motocitizen.network;

public enum Methods {
    AUTH("auth"),
    GET_LIST("list"),
    BAN("ban"),
    ON_WAY("onway"),
    CANCEL_ON_WAY("cancelOnWay"),
    IN_PLACE("inplace"),
    LEAVE("leave"),
    MESSAGE("message"),
    CREATE("createAcc"),
    REGISTER_GCM("registerGCM"),
    CHANGE_STATE("changeState");

    private final String code;

    Methods(String code) {
        this.code = code;
    }

    public String toCode() {
        return this.code;
    }
}

File: Motocitizen/src/motocitizen/network/Methods.kt
package motocitizen.network

enum class Methods constructor(val code: String) {
    AUTH("auth"),
    LIST("getList"),
    BAN("ban"),
    DETAILS("getDetails"),
    ON_WAY("onway"),
    CANCEL_ON_WAY("cancel"),
    IN_PLACE("inPlace"),
    LEAVE("leave"),
    MESSAGE("newMessage"),
    CREATE("createAccident"),
    ACTIVATE_ACCIDENT("activateAccident"),
    END_ACCIDENT("endAccident"),
    HIDE_ACCIDENT("hideAccident");
}
